### DISCUSSION TITLE
Outage to MASS

### DISCUSSION THREAD

**Discussion by mo-mahammedk** [MetOffice/simulation-systems#176] [Monsoon]
**Wednesday 1st May: Outage to MOOSE 'GET' 'SELECT' and 'FILTER' commands for system maintenance**
 
At 10:00L on Wednesday 1st May there will be an outage to MASS to allow for system maintenance. Between 10:00L and 12:00L all **PUT** command data will be cached in MOOSE and ingested after the maintenance has taken place, GET's/SELECT's and FILTER commands will fail during this period.

**Comment by mo-mahammedk** [comment]
The system is back processing commands as normal. Please note that there is a large backlog of commands to process, so things may be a little slower for a while.
 
Please also note that there will be an additional outage tomorrow:
 
**Thursday 2nd May: Outage to MOOSE 'PUT' 'GET' 'SELECT and 'FILTER' commands for system maintenance related to Microsoft migration**
 
At 10:00 BST on Thursday 2nd May there will be an outage to MASS in order to shut down the Prod database in order to take a snapshot. This is scheduled between 10:00 BST and 12:00 BST but may end sooner. All MOOSE 'PUT, 'GET', 'SELECT' and 'FILTER' commands will fail.

**Comment by mo-roger-milton** [comment]
MOOSE work is now complete, so the system is avaialbe for use again.
